resume-buddy/src/test/java/com/google/sps/PairTest.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps;
+
+import com.google.sps.data.Pair;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Tests for Pair class */
+@RunWith(JUnit4.class)
+public class PairTest {
```

Comment:
This test file should be in

`resume-buddy/test/java/com/google/sps/data/PairTest.java`

instead of

`resume-buddy/src/test/java/com/google/sps/PairTest.java`

because `src/` should only contain source code and Pair.java is in `sps/data`, so PairTest.java's location should mirror that.

Reply:
Still put this test in `resume-buddy/src/test/java/com/google/sps/data/PairTest.java` (in a data/ folder)